package-lock.json

```diff
@@ -7319,7 +7319,8 @@
         "ansi-regex": {
```

Comment:
There shouldn't be any change to the package-lock as this PR isn't modifying dependencies.